src/main/java/mate/jdbc/service/CarServiceImpl.java

```diff
@@ -0,0 +1,69 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Car get(Long id) {
+        return carDao.get(id).orElseThrow(
+                () -> new IllegalArgumentException("Car with id " + id + " not found"));
```

Comment:
```suggestion
                () -> new NoSuchElementException("Car with id " + id + " not found"));
```
тут краще цей ексепшин буде підходити, адже з аргументами все ок, просто в базі нема такого запису